### ISSUE TITLE
Tweets are no longer viewable unless logged in

### ISSUE THREAD

**Issue by ElectBitcoin** [xacnio/tweetcapture#42]
You can no longer view tweets on twitter unless you are logged in. 

In addition, there are new limits for logged in users: https://twitter.com/elonmusk/status/1675187969420828672

Is there any chance there is a way to pass a username and password to the chrome webdriver so that we can continue to screenshot tweets? I'm less concerned with the rate limit because we screenshot less than 10 tweets per week.

**Comment by ElectBitcoin** [comment]
Another option could be to utilize the twitter embed widget, which still allows for public viewing of tweets (for now, at least 🙄)

Example: https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2Felonmusk%2Fstatus%2F1675187969420828672&widget=Tweet

**Comment by xacnio** [comment]
Fixed with #43
Update to 0.2.0

Login your Twitter account on your browser and check auth_token from DevTools > Application > Cookies.

![2023-07-02_00-48-36](https://github.com/xacnio/tweetcapture/assets/49660313/e8871ea6-5041-4e1c-8652-1a2f24612a5b)

Copy auth_token and use.

## Set as environment value
AUTH_TOKEN=token_here

## Use with command line
```
tweetcapture https://twitter.com/jack/status/20 --cookies auth_token=token_here
```


## Use in your code
```python
from tweetcapture import TweetCapture
import asyncio
tweet = TweetCapture()
tweet.set_cookies([{'name': 'auth_token', 'value': 'token-here'}])
asyncio.run(tweet.screenshot("https://twitter.com/jack/status/20"))
```
